packages/playground/src/config/commonConfig.js

```diff
@@ -2,7 +2,7 @@ import getCurrency from './getCurrency';
 import { getSearchParameters } from '../utils';
 
 const DEFAULT_LOCALE = 'en-US';
-const DEFAULT_COUNTRY = 'US';
+const DEFAULT_COUNTRY = 'GB';
```

Comment:
Is this deliberate?

Reply:
Not really, I'll roll it back. Thanks for checking!